.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "challocator"]
+	path = challocator
+	url = ./challocator/
```

Comment:
This won't work, you shall specify the url of your git repository.

(Probably https://github.com/Gaspard--/chaos-2 or similar)

Reply:
should be fixed now.